samples/excel/89-preview-apis/events-worksheet-activated.yaml

```diff
@@ -0,0 +1,156 @@
+order: 7
+id: excel-events-worksheet-activated
+name: Events - Worksheet activated
+description: Add an event handler on the worksheet activated and deactivated events
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#activate-event-handler").click(() => tryCatch(onActivateEventHandler));
+        $("#deactivate-event-handler").click(() => tryCatch(onDeactivateEventHandler));
+        $("#delete-worksheet").click(() => tryCatch(deleteWorksheet));
+
+        async function onActivateEventHandler() {
+            await Excel.run(async (context) => {
+                let sheet = context.workbook.worksheets;
+                sheet.onActivated.add(onActivate);
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("Registered worksheet on activated event handler.");   
```

Comment:
The string "Registered worksheet on activated event handler."  has a little too much geek-syntax to be readable. I recommend "A handler has been registered for the OnActivated event."